### PR TITLE
initial admin runbooks

### PR DESCRIPTION
**Pull request by kensipe**
Fixes: #150 
Signed-off-by: Ken Sipe <kensipe@gmail.com>

### CODE REVIEW

**Comment by gerred** [comment]
the content lgtm, but I'm thinking these should be a bunch of different pages to keep it short, make each slug searchable, etc.

File: content/docs/runbooks/admin/add-operator-to-repository.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### Build an Index file
+
+```
+kubectl kudo repo index ~/repo --merge-repo community --url-repo communit
+```
+This will download the current index from the community repository.  Add all operators in the `~/repo` folder to the index using the community url as the base location this operator will be hosted.
+
+### Copy Artifacts to Repository
+
+Copy all artifacts in the `~/repo` to the serve.
```
Comment (ANeumann82) [inline content/docs/runbooks/admin/add-operator-to-repository.md]:
```suggestion
Copy all artifacts in the `~/repo` to the server.
```

File: content/docs/runbooks/admin/add-operator-to-repository.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+* You have read/write access to a web based server which hosts a respository.
+* You have an operator package built (this runbook assumes the first-operator in ~/repo folder)
+
+## Steps
+
+### Build an Index file
+
+```
+kubectl kudo repo index ~/repo --merge-repo community --url-repo communit
```
Comment (ANeumann82) [inline content/docs/runbooks/admin/add-operator-to-repository.md]:
```suggestion
kubectl kudo repo index ~/repo --merge-repo community --url-repo community
```

File: content/docs/runbooks/admin/create-kudo-package.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+A KUDO operator has been created.  This runbook uses the [first-operator operator](https://github.com/kudobuilder/operators/tree/master/repository/first-operator) in the [operators github repository](https://github.com/kudobuilder/operators).  It is expected that the working directory is from the base of the operator project.
+
+## Steps
+
+### Package KUDO Operator
+
+```
+rm -rf ~/repo
+mkdir ~/repo
```
Comment (ANeumann82) [inline content/docs/runbooks/admin/create-kudo-package.md]: Could we replace this with `mkdir -p ~/repo`? 

On one hand, I think users doing this should be smart enough to not rm a directory that they actually use, but on the other hand... Who knows who's following these guides

File: content/docs/runbooks/admin/create-operator.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+This creates the main structure of the operator which can be view with `tree`
+
+```
+tree .
+.
+└── operator
+    ├── operator.yaml
+    └── params.yaml
+```
```
Comment (ANeumann82) [inline content/docs/runbooks/admin/create-operator.md]: I don't think `tree` is available by default on MacOs, not sure about linux. Maybe rather use `ls -aR` or something similar?

File: content/docs/runbooks/admin/remove-kudo.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+## Preconditions
+
+KUDO is initialized and running on a cluster.
+
+## Steps
+
+### Uninstall KUDO
+
+`kubectl kudo init --dry-run --output yaml | kubectl delete -f-`
+
```
Comment (ANeumann82) [inline content/docs/runbooks/admin/remove-kudo.md:14]: Should we add a warning that this removes all installed operators and deployments as well?

File: content/docs/runbooks/admin/create-kudo-package.md (hunk 3 above)
Comment (ANeumann82) [reply content/docs/runbooks/admin/create-kudo-package.md]: Maybe we can add a warning? Mmmm.

File: content/docs/runbooks/admin/add-operator-to-repository.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+`kubectl kudo repo index ~/repo --merge-repo community --url-repo community`
+
+This will download the current index from the community repository.  Add all operators in the `~/repo` folder to the index using the community url as the base location this operator will be hosted.
+
+### Copy Artifacts to Repository
+
+Copy all artifacts in the `~/repo` to the serve.
+
+::: tip Note
+When updating a repository it is best to copy over all operator packages, then update the index file.
```
Comment (ANeumann82) [inline content/docs/runbooks/admin/add-operator-to-repository.md]:
```suggestion
When updating a repository it is best to copy over all operator packages first, then update the index file.
```

**Review by ANeumann82 — APPROVED** [review]

**Comment by kensipe** [comment]
thanks @mattj-io I've incorporated your feedback....   Appreciate it!

**Pull request merged**